examples/flutter_gallery/lib/demo/material/dialog_demo.dart

```diff
@@ -122,6 +123,7 @@ class DialogDemoState extends State<DialogDemo> {
               showDemoDialog<DialogDemoAction>(
                 context: context,
                 child: new AlertDialog(
+                  routeName: 'location permissions',
```

Comment:
why lowercase?

Reply:
(as opposed to `Location permissions.`, say)

Reply:
Fixed